keyboardvisibilityevent/src/main/java/net/yslibrary/android/keyboardvisibilityevent/KeyboardVisibilityEvent.kt

```diff
@@ -14,6 +18,23 @@ private const val KEYBOARD_MIN_HEIGHT_RATIO = 0.15
  */
 object KeyboardVisibilityEvent {
 
+    @JvmStatic
+    fun setEventListener(
```

Comment:
Could you add documentation for this method?

Reply:
Documentation added